doc/getting-started/getting-started.md

```diff
@@ -0,0 +1,804 @@
+The relationship between them is not shareable with others; it is unique to those two parties in that each pairwise relationship uses different DIDs. 
+We call process of establish connection **Onboarding**. 
+In this tutorial we will describe the simple version of onboarding process. 
+In our case one party will be always Trust Anchor. Real enterprise scenarios can use more complex version. 
+
+##### Connection establishment
+Let's look the process of connection establishment between **Steward** and **Faber College**.
+
+1. **Faber** and **Steward** contact in a some way to initiate onboarding process. 
+   It can be filling the form on web site or phone call.
+1. **Steward** creates new DID record in wallet by calling ``did.create_and_store_my_did`` that he will use for secure interactions with **Faber**.
+1. **Steward** sends corresponding `NYM` transaction to the Ledger by calling consistently ``ledger.build_nym_request`` to build NYM request and ``ledger.sign_and_submit_request`` to send the created request.
+1. **Steward** creates connection request which contains created `DID` and `Nonce`. 
+   This nonce is just a big random number generated to track the unique connection request. 
+   A nonce is a random arbitrary number that can only be used one time.  
+   When a connection request is accepted, the invitee digitally signs the nonce such that the inviter can match the response with a prior request.
+1. **Steward** sends connection request to **Faber**.
+1. **Faber** accepts connection request from **Steward**.
+1. **Faber** creates wallet if it does not exist yet.
+1. **Faber** creates new DID record in his wallet by calling ``did.create_and_store_my_did`` that he will use for secure interactions with **Steward**.
+1. **Faber** creates connection response which contains created `DID`, `Verkey` and `Nonce` from received connection request.
+1. **Faber** asks ledger for Verification key of **Steward's** DID by calling ``did.key_for_did``.
+1. **Faber** anonymous encrypts connection response by calling ``crypto.anon_crypt`` with **Steward** Verkey.
+   Anonymous-encryption schema is designed for sending of messages to a Recipient given its public key. 
+   Only the Recipient can decrypt these messages, using its private key. 
+   While the Recipient can verify the integrity of the message, it cannot verify the identity of the Sender.
+1. **Faber** sends anonymous encrypted connection response to **Steward**.
+1. **Steward** anonymous decrypts connection response by calling ``crypto.anon_decrypt``.
+1. **Steward** authenticates **Faber** by comparision of Nonce.
+1. **Steward** sends `NYM` transaction for **Faber's** DID to the Ledger. 
+Please note that despite Steward is sender of this transaction the owner of DID will be Faber as it uses Verkey provided by Faber.
+
+```python
+  (steward_faber_did, steward_faber_key) = await did.create_and_store_my_did(steward_wallet, "{}")
```

Comment:
We need to add the comment about the agent that executes each code piece for each code piece included.